drivers/SmartThings/zigbee-contact/src/init.lua

```diff
@@ -0,0 +1,32 @@
+-- Copyright 2021 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local capabilities = require "st.capabilities"
+local ZigbeeDriver = require "st.zigbee"
+local constants = require "st.zigbee.constants"
+local defaults = require "st.zigbee.defaults"
+
+local zigbee_contact_driver_template = {
+  supported_capabilities = {
+    capabilities.contactSensor,
+    capabilities.temperatureMeasurement,
+    capabilities.battery,
+  },
+  sub_drivers = { require("battery-overrides") },
```

Comment:
Does it not work to use a bare string here over the wrapped parens?

Reply:
Not certain what you're asking. This is the standard way of including sub-drivers

Reply:
I believe @FreeMasen is asking if it could just be written as `require "battery-overrides"` here, which I believe it could.  I don't think we have a strict policy for omission of parens for function calls or `require`s specifically so I would say we leave this as-is apart from any clear standard.

Reply:
Since the other `require` calls in this file do not use the parenthesis I was curious why this would be different. Regardless of style, it is typical to try and keep usage consistent internally for a file.